README.md

```diff
@@ -16,17 +34,55 @@ Generated documentation for this SDK can be found at https://rakutentech.github.
 
 To run the example project, clone the repo, and run `pod install` from the Example directory first.
 
-## Requirements
+### Custom parameters
+
+In order to load your MiniApp, you will have to use your own Host App ID and your subscription key. These can eiher be set in project configuration plist (RASApplicationIdentifier and RASProjectSubscriptionKey) or by taping the top right configuration icon in the example application.
 
 ## Installation
 
-MiniApp is available through [CocoaPods](https://cocoapods.org). To install
-it, simply add the following line to your Podfile:
+MiniApp is available through [CocoaPods](https://cocoapods.org). To install it, simply add the following line to your Podfile:
 
 ```ruby
 pod 'MiniApp'
 ```
 
+In your project configuration .plist you can add 2 custom string target properties :
+
+- `RASApplicationIdentifier` - to set your MiniApp host application identifier
+- `RASProjectSubscriptionKey` - to set your MiniApp subsbscription key
```

Comment:
Also should list the base url property here.

Reply:
done